docs/source/config_files.rst

```diff
@@ -126,12 +126,18 @@ their demo for binder <https://github.com/jupyterlab/jupyterlab-demo/blob/master
 ``start``
 ^^^^^^^^^
 
-A script that can contain arbitrary commands to be run at runtime (as an
+A script that can contain simple commands to be run at runtime (as an
```

Comment:
Try and suggest that you can only do simple things here in an attempt to keep people from doing heavy computations, even if we do not enforce it.